geometry/SettingsViewController.swift

```diff
@@ -8,20 +8,18 @@
 
 import UIKit
 
-protocol SettingsViewControllerDelegate: class {
-    func settingsViewControllerDelegate(sender: SettingsViewController)
+protocol SettingsViewControllerDelegate {
+    func fillSettingsViewControllerDelegate(rColor: Float, gColor: Float, bColor: Float)
```

Comment:
'setColor' method name would tell just enough information. 'fillSettingsViewControllerDelegate' is too long.